.env.example.cjs

```diff
@@ -66,10 +67,9 @@ process.env.GITLAB_ACCESS_TOKEN_USERNAME ??= "placeholder"
 // process.env.PIPELINE_SCRIPTS_REF ??= "placeholder"
 
 /*
-  This private key's file can be generated and downloaded from
+  Download private key file from
   https://github.com/settings/apps/[app-name].
-  If you need to calculate the Base64 of the value manually, that can be done
```

Comment:
The text is saying **If you need**. Quite explicitly it's not needed in this case because the conversion is done by `Buffer.from`. I don't mind this being removed but I don't think it's necessary to remove it.